### ISSUE TITLE
meter stops recording halfway through a fight

### ISSUE THREAD

**Issue by integrax3** [snoww/loa-logs#36]
it was working fine before patch on 9/20

after patch, now it would record halfway through a fight and stop(dps will decline over time) so it seems to still be running but no new data coming in.

I have already tried to reinstall it

maybe helpful info: LOA details works without any issue

thanks in advance!

**Comment by snoww** [comment]
what is your setup? VM or 1PC? any VPNs?

**Comment by Quiksmage** [comment]
if using vm:
this happened to me with virtual box as my hypervisor. Hyper-v doesn't have this issue.

**Comment by integrax3** [comment]
i am using VM, vmware, no vpn
the funny thing is, hyperv never worked for me with loa log which is why i switched to vmware.
but its strange since loa details works completely fine. it was only after the patch 2 weeks ago where this issue developed

**Comment by Quiksmage** [comment]
they use different packet sniffers. I had the same experience with loa details working fine. i'd try hyper-v again if you want to use this meter in a vm. or a different hypervisor. maybe kvm would work too. what issue did you have running it in hyper-v? if you got it running, but no capture, you need to setup the nic properly.

**Comment by integrax3** [comment]
> they use different packet sniffers. I had the same experience with loa details working fine. i'd try hyper-v again if you want to use this meter in a vm. or a different hypervisor. maybe kvm would work too. what issue did you have running it in hyper-v? if you got it running, but no capture, you need to setup the nic properly.

yes i had issue setting up NIC in bridge, i can give it another go, do you mind sharing how you setup the NIC?

**Comment by Quiksmage** [comment]
I followed this guide. You should only have 1 virtual nic on the VM bridged to the nic on your host machine 
https://learn.microsoft.com/en-us/azure/defender-for-iot/organizations/traffic-mirroring/configure-mirror-hyper-v

**Comment by integrax3** [comment]
> I followed this guide. You should only have 1 virtual nic on the VM bridged to the nic on your host machine https://learn.microsoft.com/en-us/azure/defender-for-iot/organizations/traffic-mirroring/configure-mirror-hyper-v
> 
> NIC settings on the VM ![image](https://user-images.githubusercontent.com/12449533/273343310-4feab7aa-a775-4dda-9039-faf83259e096.png)
> 
> Settings in the Virtual Switch settings ![image](https://user-images.githubusercontent.com/12449533/273343348-783ff866-1961-4879-aaa8-3fb7d8a74458.png)

thank you for this, finally got around to setting this up. it runs now but for some reason, every so often. like 1/5 of the time, it still stops working (dps doesnt record but the timer is still running) any ideas?
